### DISCUSSION TITLE
Request: Please add Cloud storage support to uBOL for manual backup and restore of uBOL settings

### DISCUSSION THREAD

**Discussion by awesomeocelot** [uBlockOrigin/uBOL-home#176] [Ideas]
This is a feature of your original extension https://github.com/gorhill/uBlock/wiki/Cloud-storage and imo it's quite nice to be able to manually synchronizing the configured extension settings, such as backing up and restoring the "Filter lists" and "Trusted sites" options that the user has selected. For uBOL cloud storage could be used for settings such as the "No filtering" hostnames list, websites with overridden filtering options that differ from the default filtering mode, and the specific Filter lists that a user has enabled. I also think it would be nice if each of these settings should be able to be manually backed up and restored individually, similar to the original uBlock Origin extension.

**Comment by cevdetayaz** [comment]
Hi there, i have been using uBoL on 3 devices. I have to manually add websites to white-list on all devices again. Is it possible to bring back backup options which is available on regular Ublock.  Sync option would be much better than just backup.

**Comment by gorhill** [comment]
https://github.com/uBlockOrigin/uBOL-home/issues/273
